routes/index.js

```diff
@@ -1,7 +1,23 @@
-import product from './product';
+import signUp from './auth/signUp';
+import signIn from './auth/signIn';
+import logout from './auth/logout';
+import refreshToken from './auth/refreshToken';
+import facebook from './auth/facebook';
+import google from './auth/google';
 import users from './users';
+import product from './product';
+import city from './city';
+
+import isAuth from 'middlewares/isAuth';
 
 export default (app) => {
-	app.use('/api/products', product);
-	app.use('/api/users', users);
+	app.use('/api/signup', signUp);
+	app.use('/api/signin', signIn);
+	app.use('/api/logout', isAuth, logout);
```

Comment:
For better view, please, move the routes which doesn't need `isAuth` to one block, then apply `app.use('/api', isAuth)` and then put other routes which need this verification like
```javascript
// routes which don't need verification
app.use('/api/signup', signUp);
................
// verification middleware application
app.use('/api', isAuth);
// verification-demanded routes
app.use('/api/products',product);
```

Reply:
Done.